internal/dag/builder_test.go

```diff
@@ -15936,6 +15977,18 @@ func prefixroute(prefix string, first *Service, rest ...*Service) *Route {
 	}
 }
 
+func timeoutsHTTPProute(prefix string, timeouts *gatewayapi_v1.HTTPRouteTimeouts, first *Service, rest ...*Service) *Route {
+	services := append([]*Service{first}, rest...)
+	rtp, rp, _ := parseHTTPRouteTimeouts(timeouts)
```

Comment:
hm, I don't feel great about using a helper that is being tested to generate the expected output for it's test, lets do this differently so we're not using real code here